jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/OverlaysTests.java

```diff
@@ -66,26 +56,31 @@ public void darkThemeOverlaysTest() {
         absoluteOverlayButton.show();
         absoluteOverlayButton.click();
         absoluteOverlay.has().darkTheme();
+        absoluteOverlay.close();
     }
 
     @Test(description = "Check overlay opacity value")
     public void opacityOverlaysTest() {
         opacityOverlayButton.show();
+        waitCondition(() -> opacityOverlayButton.core().isClickable());
```

Comment:
We can wait until Overlay is displayed, clickable looks very strange

Reply:
Changed to wait for Overlay.isDisplayed